spec/features/viewing_peeps_spec.rb

```diff
@@ -0,0 +1,15 @@
+feature 'Viewing peeps' do
+  scenario 'A user can see peeps in reverse chronological order' do
```

Comment:
I do not think the order of the peeps is being tested here, just that they are on the page.